include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -492,59 +492,26 @@ struct __parallel_transform_scan_static_single_group_submitter<_Inclusive, _Elem
                     const ::std::uint16_t __subgroup_id = __subgroup.get_group_id();
                     const ::std::uint16_t __subgroup_size = __subgroup.get_local_linear_range();
 
-#if _ONEDPL_SYCL_SUB_GROUP_LOAD_STORE_PRESENT
-                    constexpr bool __can_use_subgroup_load_store =
-                        _IsFullGroup && oneapi::dpl::__internal::__range_has_raw_ptr_iterator_v<::std::decay_t<_InRng>>;
-#else
-                    constexpr bool __can_use_subgroup_load_store = false;
-#endif
-
                     auto __lacc_ptr = __dpl_sycl::__get_accessor_ptr(__lacc);
-                    if constexpr (__can_use_subgroup_load_store)
-                    {
-                        _ONEDPL_PRAGMA_UNROLL
-                        for (::std::uint16_t __i = 0; __i < _ElemsPerItem; ++__i)
-                        {
-                            auto __idx = __i * _WGSize + __subgroup_id * __subgroup_size;
-                            auto __val = __unary_op(__subgroup.load(__in_rng.begin() + __idx));
-                            __subgroup.store(__lacc_ptr + __idx, __val);
-                        }
-                    }
-                    else
+                    for (::std::uint16_t __idx = __item_id; __idx < __n; __idx += _WGSize)
```

Comment:
Can we change all of the `::std` instances to `std` in the affected lines of code?

Reply:
Done